src/Organizer/EditOrganizerRestController.php

```diff
@@ -80,6 +83,101 @@ public function create(Request $request)
     }
 
     /**
+     * @param $organizerId
+     * @param Request $request
+     * @return JsonResponse
+     * @throws DataValidationException
+     */
+    public function updateWebsite($organizerId, Request $request)
+    {
+        $websiteJSONDeserializer = new WebsiteJSONDeserializer();
+        $website = $websiteJSONDeserializer->deserialize(
+            new StringLiteral($request->getContent())
+        );
+
+        try {
+            $commandId = $this->editingService->updateWebsite(
+                $organizerId,
+                $website
+            );
+        } catch (UniqueConstraintException $e) {
+            $e = new DataValidationException(
+                [
+                    'website' => 'Should be unique but is already in use.',
+                ]
+            );
+            throw $e;
+        }
+
+        return JsonResponse::create(['commandId' => $commandId]);
+    }
+
+    /**
+     * @param $organizerId
+     * @param Request $request
+     * @return JsonResponse
+     * @throws DataValidationException
+     */
+    public function updateTitle($organizerId, Request $request)
+    {
+        $titleJSONDeserializer = new TitleJSONDeserializer();
```

Comment:
Als we in de Swagger docs "title" vervangen met "name", dan zouden we hier beter een NameJSONDeserializer gebruiken. We kunnen die dan later ook herbruiken als we de offer endpoints opkuisen. Dan kunnen we TitleJSONDeserializer daarna ook verwijderen uit udb3-php. Die hoort daar sowieso niet thuis.